### ISSUE TITLE
[BUG] Error: Invalid `client`. `client` parameter is required

### ISSUE THREAD

**Issue by MadeGOD** [wonderlandpark/dokdo#66]
```sh
0|bot    | Error: Invalid `client`. `client` parameter is required.
0|bot    |     at new Dokdo (/root/DM-support-bot/node_modules/dokdo/src/index.js:39:52)
0|bot    |     at Object.<anonymous> (/root/DM-support-bot/bot.js:147:22)
0|bot    |     at Module._compile (internal/modules/cjs/loader.js:999:30)
0|bot    |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
0|bot    |     at Module.load (internal/modules/cjs/loader.js:863:32)
0|bot    |     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
0|bot    |     at Object.<anonymous> (/usr/local/lib/node_modules/pm2/lib/ProcessContainerFork.js:33:23)
0|bot    |     at Module._compile (internal/modules/cjs/loader.js:999:30)
0|bot    |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
0|bot    |     at Module.load (internal/modules/cjs/loader.js:863:32)
```

file bot.js

```js
/*
        독도 시스템
*/
const Dokdo = require('dokdo')

const Discord = require("discord.js")
const client = new Discord.Client()
const DokdoHandler = new Dokdo(client, { aliases: ['dokdo', 'dok'], prefix: '//' }) // Using Bot Application ownerID as default for owner option.

client.on('message', async message => {
  if (message.content === 'ping') return message.channel.send('Pong') // handle commands first
  DokdoHandler.run(message) // try !dokdo
})
```

**Comment by MadeGOD** [comment]
```js
/*
        Using for Discord.js, fs, Child Process

*/

const { inspect } = require('util')
const Discord = require("discord.js")
const client = new Discord.Client()
const fs = require("fs");
const settings = require('./config.json')
const { prefix } = require('./config.json')
const restart = require('./restart.json');
/*
        독도 시스템
*/
const Dokdo = require('dokdo')

const DokdoHandler = new Dokdo(client, { aliases: ['dokdo', 'dok'], prefix: '//' }) // Using Bot Application ownerID as default for owner option.

client.on('message', async message => {
  if (message.content === 'ping') return message.channel.send('Pong') // handle commands first
  DokdoHandler.run(message) // try !dokdo
})
```

**Comment by MadeGOD** [comment]
@wonderlandpark

**Comment by MoartMedia** [comment]
![image](https://user-images.githubusercontent.com/18746638/104996568-73754d80-5a6b-11eb-8516-50f25c04e502.png)

Can Reproduce. (Dokdo version: 0.1.0)

It maybe conflict Discord.JS with other version when check `Discord.Client`.
so must change to optional Dependeries.

**Comment by wonderlandpark** [comment]
Duplicate of #45
Fixed in #52
